components/ai_chat/core/browser/engine/engine_consumer_oai.cc

```diff
@@ -191,9 +192,25 @@ void EngineConsumerOAIRemote::OnGenerateQuestionSuggestionsResponse(
     return;
   }
 
-  std::vector<std::string> questions =
-      base::SplitString(*result, "|", base::WhitespaceHandling::TRIM_WHITESPACE,
-                        base::SplitResult::SPLIT_WANT_NONEMPTY);
+  constexpr char kStartTag[] = "<question>";
+  constexpr char kEndTag[] = "</question>";
+
+  base::StringTokenizer tokenizer(*result, "\n");
```

Comment:
we should assume they are separated by "\n". They could be inline.

Reply:
improved parser